ocaml/xapi/xapi_globs.ml

```diff
@@ -1513,6 +1517,30 @@ let other_options =
     , (fun () -> string_of_bool !observer_endpoint_https_enabled)
     , "Enable https endpoints to be used by observers"
     )
+  ; ( "observer-experimental-components"
+    , Arg.String
+        (fun s ->
+          observer_experimental_components :=
+            match s with
+            | "" ->
+                StringSet.empty
+            | s ->
+                let input_set =
+                  s |> String.split_on_char ',' |> StringSet.of_list
+                in
+                let valid_set =
+                  Constants.observer_components_all |> StringSet.of_list
+                in
+                StringSet.inter input_set valid_set
+        )
+    , (fun () ->
+        !observer_experimental_components
+        |> StringSet.to_seq
+        |> List.of_seq
```

Comment:
this works, but you can use `StringSet.elements` here to get a list directly and make this a little bit shorter.